notebook/i18n/ru_RU/LC_MESSAGES/nbjs.po

```diff
@@ -83,7 +83,7 @@ msgstr "Применить"
 #: notebook/static/base/js/dialog.js:225
 msgid "WARNING: Could not save invalid JSON."
 msgstr "ПРЕДУПРЕЖДЕНИЕ: не удалось сохранить недопустимый JSON."
-
+12
```

Comment:
Just small question here @andrii-i, this was a change that seemed new... I have no knowledge of Russian but since it was the only number added I wanted to double check with you in case it may be a typo?

Reply:
Oh it's a total typo. Thank you for catching this

Reply:
Removed it